src/app/components/Item/ItemHoldings.jsx

```diff
@@ -55,7 +55,7 @@ class ItemHoldings extends React.Component {
   /*
    * getRecord(e, bibId, itemId)
    * @description Get updated information for an item along with its delivery locations,
-   * and route to the correct page.
+   * and the route to the correct page.
```

Comment:
Ohh so in this case I was using `route` as a verb so I meant that it would "go to the correct page". Adding `the` is fine.